src/Bootstraps/DiscoveryLocationBootstrap.php

```diff
@@ -32,7 +29,7 @@ public function boot(): void
 
     private function discoverInstalledPackageLocations(): array
     {
-        $composerPath = path($this->root, 'vendor/composer');
+        $composerPath = path($this->kernel->root, 'vendor/composer');
```

Comment:
I wish we could refactor this somehow. Does this belong in `AppConfig`?

Reply:
https://github.com/tempestphp/tempest-framework/issues/187